### PR TITLE
utils_libvirt/libvirt_disk: add function to get disk not on /

### PR DESCRIPTION
**Pull request by smitterl**
Use "/" mount information to identify a disk and return its name.

### CODE REVIEW

**Review by chunfuwen — APPROVED** [review]
lgtm

File: virttest/utils_libvirt/libvirt_disk.py [hunk 1]
```diff
@@ -24,6 +24,83 @@
...
+            continue
+        name, mpoint = entry.groups()
+        is_disk = disk_pattern.match(name)
+        if root_mounted:
+            if is_disk:
+                root_disk = line
+                break
+            else:
+                idx = idx - 1
+                continue
```
Comment (nanli1) [inline virttest/utils_libvirt/libvirt_disk.py:81]: Just confirm, We will never run into this `if root_mounted` branch if we set  ` root_mounted = False` in L63 and not pass it in `def get_non_root_disk_name(session)` right?

Comment (smitterl) [reply virttest/utils_libvirt/libvirt_disk.py:81]: @nanli1 That is correct this branch will not be entered if root_mounted is never set to True in L. 83. This means there is no mount point for "/" (L. 82), therefore the idx will just go up until len(lines) and the TestError is raised  (L. 94).

**Comment by smitterl** [comment]
In the last push I modified the following lines to remove whitespace from the return values
```
        name, mpoint = entry_pattern.match(non_root_disks[0]).groups()
        return name.strip(), mpoint.strip()
```

This was necessary because the libvirt_disk.linux_disk_check was failing because the name was returned as 'vdb             ' with additional whitespace. This is fixed now and doesn't invalidate previous test results as they assumed this was already the case.

**Review by nanli1 — APPROVED** [review]

**Pull request merged**